src/makers/win32/wix.js

```diff
@@ -0,0 +1,30 @@
+import path from 'path';
+
+import { ensureDirectory } from '../../util/ensure-output';
+import configFn from '../../util/config-fn';
+import isInstalled from '../../util/is-installed';
+import { getDistinguishedNameFromAuthor } from './appx';
+
+export const isSupportedOnCurrentPlatform = async () => isInstalled('electron-wix-msi');
+
+export default async ({ dir, appName, targetArch, forgeConfig, packageJSON }) => {
+  const { MSICreator } = require('electron-wix-msi');
+
+  const outPath = path.resolve(dir, `../make/wix/${targetArch}`);
+  await ensureDirectory(outPath);
+
+  const creator = new MSICreator(Object.assign({
+    description: packageJSON.description,
+    name: appName,
+    version: packageJSON.version,
+    manufacturer: getDistinguishedNameFromAuthor(packageJSON.author).substr(3),
```

Comment:
We should probably have a separate function that doesn't have `CN=` prepended to it, because I was confused why `.substr(3)` was there.